src/encord_active/cli/print.py

```diff
@@ -33,7 +33,9 @@ def print_encord_projects(
     project_query = None if query is None else ProjectQuery(title_like=query)
     json_projects = get_projects_json(app_config.get_or_query_ssh_key(), project_query)
     if state.get("json_output"):
-        Path("./encord-projects.json").write_text(json_projects, encoding="utf-8")
+        output_file_path = Path("./encord-projects.json")
```

Comment:
no resolve() here?

also the sequence: path.expanduser().absolute().resolve() is the best bet to have the project output a canonical path.

Reply:
good catch with the resolve